pkgs/applications/office/tusk/default.nix

```diff
@@ -36,7 +37,7 @@ in appimageTools.wrapType2 rec {
   multiPkgs = null; # no 32bit needed
   extraPkgs = appimageTools.defaultFhsEnvArgs.multiPkgs;
   extraInstallCommands = ''
-    mv $out/bin/{${name},${pname}}
+    mv $out/bin/{${pname}-${version},${pname}}
     mkdir "$out/share"
```

Comment:
The name has a `-v` instead of a `-`, is it correct that it wasn't moving the correct file before?

Reply:
The build fails if I add it back, so it must have been broken already.

Reply:
Nice fix then!